flask_rebar/rebar.py

```diff
@@ -8,21 +8,39 @@
     :copyright: Copyright 2018 PlanGrid, Inc., see AUTHORS.
     :license: MIT, see LICENSE for details.
 """
+from __future__ import annotations
```

Comment:
this is for type hints like `def copy(self) -> InstanceOfThisClass` where I'm referencing a class from inside it

Reply:
Which Python versions is this needed for? (Is this only required for 3.8 support?)

Reply:
here it says TBD on when this will become mandatory https://docs.python.org/3/library/__future__.html